src/etc/inc/ipsec.inc

```diff
@@ -1668,18 +1668,41 @@ function ipsec_setup_bypasslan() {
 	if (!empty($bypassnets)) {
 		$bypass = implode(',', $bypassnets);
 		$scconf['connections']['bypass'] = array();
-		/* Prevents the connection from being considered for remote peers */
-		$scconf['connections']['bypass']['remote_addrs'] = "127.0.0.1";
 		$scconf['connections']['bypass']['children'] = array();
-		$scconf['connections']['bypass']['children']['bypass'] = array();
-		$scconf['connections']['bypass']['children']['bypass']['local_ts'] = $bypass;
-		$scconf['connections']['bypass']['children']['bypass']['remote_ts'] = $bypass;
-		$scconf['connections']['bypass']['children']['bypass']['mode'] = "pass";
-		$scconf['connections']['bypass']['children']['bypass']['start_action'] = "trap";
+		$scconf['connections']['bypass']['children']['bypasslan'] = array();
+		$scconf['connections']['bypass']['children']['bypasslan']['local_ts'] = $bypass;
+		$scconf['connections']['bypass']['children']['bypasslan']['remote_ts'] = $bypass;
+		$scconf['connections']['bypass']['children']['bypasslan']['mode'] = "pass";
+		$scconf['connections']['bypass']['children']['bypasslan']['start_action'] = "trap";
 	}
 	return;
 }
 
+function ipsec_setup_extrabypass() {
```

Comment:
Why make an extra function?

If they are both only ever called together, just use one function since it's all setting up the same section.